security/signature_validation/signature_validation.1.x.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/twilio/twilio-go/client"
+)
+
+func main() {
+	// You can find your Auth Token at twilio.com/console
+	// For this example: authToken := "12345"
+	authToken := os.Getenv("TWILIO_AUTH_TOKEN")
+
+	requestValidator := client.NewRequestValidator(authToken)
+
+	// Twilio's request URL
+	url := "https://mycompany.com/myapp.php?foo=1&bar=2"
```

Comment:
```suggestion
	url := "https://example.com/myapp.php?foo=1&bar=2"
```

Reply:
Is it possible to use more realistic query params (to provide readers some context about why you'd use them)?